### PR TITLE
add iamgunjan

### LINKED ISSUE #642: Gunjan Paneri
Screenshot (938)

### PR DESCRIPTION
**Pull request by iamgunjan**
resolve #642

### CODE REVIEW

**Review by AvidCoder101 — CHANGES_REQUESTED** [review]

File: data.json [hunk 1]
```diff
@@ -1246,5 +1246,10 @@
...
     "imageUrl": "https://user-images.githubusercontent.com/66031743/128864527-177a97f6-967c-4b18-93d2-c02472274612.PNG",
     "issueNumber": 639
-  }
+  },
+  {
+    "name": "Gunjan Paneri",
+    "githubUsername": "iamgunjan",
+    "imageUrl": "https://user-images.githubusercontent.com/53567221/129094169-e012e3fb-3ba9-47e7-9173-0e09e7517034.png",
+    "issueNumber": 642
 ]
```
[the remark this answers is not in the document]
Comment (schmelto) [reply data.json:1255]: The `] ` is right the but there is an additional missing `} ` closing tag

File: data.json [hunk 2]
```diff
@@ -1246,5 +1246,10 @@
     "githubUsername": "bhagatpratham",
     "imageUrl": "https://user-images.githubusercontent.com/66031743/128864527-177a97f6-967c-4b18-93d2-c02472274612.PNG",
     "issueNumber": 639
-  }
+  },
+  {
+    "name": "Gunjan Paneri",
+    "githubUsername": "iamgunjan",
+    "imageUrl": "https://user-images.githubusercontent.com/53567221/129094169-e012e3fb-3ba9-47e7-9173-0e09e7517034.png",
+    "issueNumber": 642
```
Comment (schmelto) [inline data.json:1254]:
```suggestion
    "issueNumber": 642
  }
```
I think it should look something like this

**Review by schmelto — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: data.json [hunk 3]
```diff
@@ -1246,5 +1246,11 @@
...
+  },
```
Comment (schmelto) [inline data.json:1255]:
```suggestion
  }
```

**Review by schmelto — CHANGES_REQUESTED** [review]

**Review by schmelto — APPROVED** [review]

**Comment by schmelto** [comment]
Thanks for your contribution 🚀

**Pull request merged**